Data-Science/MLflow/bike-sharing-mlflow.ipynb

```diff
@@ -27,7 +27,65 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Set Expriment"
+    "## User Authentication"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "EZAF_ENV = \"hpe-staging-ezaf\"\n",
+    "token_url = f\"https://keycloak.{EZAF_ENV}.com/realms/UA/protocol/openid-connect/token\"\n",
+    "\n",
+    "data = {\n",
+    "    \"username\" : input(\"Enter username.\"),\n",
+    "    \"password\" : input(\"Enter password.\"),\n",
+    "    \"grant_type\" : \"password\",\n",
+    "    \"client_id\" : \"ua-grant\",\n",
+    "}\n",
+    "\n",
+    "token_responce = requests.post(token_url, data=data, allow_redirects=True, verify=False)\n",
+    "\n",
+    "token = token_responce.json()[\"access_token\"]"
```

Comment:
We can get jwt token through os.environ variable, we don't need to call keycloak api.